nautobot/project-static/css/base.css

```diff
@@ -1,517 +1,544 @@
 /* Layout */
 * {
-    margin: 0;
+  margin: 0;
```

Comment:
Can you please revert the indentation/whitespace changes? Makes for a lot of noise in the diffs.

Reply:
My mistake, that's the auto-formatter at work.  I'll get this reverted.

Reply:
Would still like to see this reverted. :-)

Reply:
My bad, I thought I did this already.

Reply:
Fixed